.editorconfig

```diff
@@ -13,5 +13,3 @@ end_of_line = lf
 insert_final_newline = true
 trim_trailing_whitespace = true
 charset = utf-8
-indent_style = space
```

Comment:
No, my best advice would be to leave this as indet_style tabs and let the user choose what to render tabs as if they prefer tabs or spaces and that way they can decide if they want 4 spaces or 2 spaces spacing.